x-pack/plugins/security_solution/public/alerts/pages/detection_engine/rules/details/index.tsx

```diff
@@ -5,6 +5,7 @@
  */
 
 /* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable complexity */
```

Comment:
NOTE: This file is due for a refactor. This is already on our list of to dos so this should hopefully only be a very temporary workaround.

Reply:
I'm pretty sure @FrankHassanabad has email alerts for anytime code goes in with this `eslint-disable` 😉  Just kidding, haha -- really though, this has saved us so many times in testing, so thanks for the note here, but may want to add a `TODO` with your comment for passerby's: 

```suggestion
// eslint-disable complexity TODO: Temporary till this component is refactored as part of lists UI integration
```

Reply:
haha I know, I know 😅  I wouldn't risk it, except for the fact that we'll have to refactor the file for lists so it won't be missed.